projects/components/src/button/button.component.scss

```diff
@@ -188,6 +188,22 @@
   }
 }
 
+.small-square {
+  height: 32px;
+  width: 32px;
+  min-width: auto;
```

Comment:
Is it only different from `small` class by width and min-width? Can we achieve the same result with the `small` class?

Reply:
After implementing the-toggle-button-group instead of ht-button I removed all changes in button component and defined styles for pagination buttons in parent component